src/OrchardCore.Modules/OrchardCore.ContentFields/OrchardCore.ContentFields.csproj

```diff
@@ -4,16 +4,22 @@
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
     <!-- NuGet properties-->
     <Title>OrchardCore Content Fields</Title>
-    <Description>$(OCCMSDescription)
+    <Description>
+      $(OCCMSDescription)
 
-    Provides features to add content fields and indexing them for a content type.</Description>
+      Provides features to add content fields and indexing them for a content type.
+    </Description>
```

Comment:
If there was no LF it's because we don't want one in the NuGet package. Unless all other packages have it.

Reply:
I'll remove it. VS auto format this on save every time.